### PR TITLE
feat: check for executable config and directory

### LINKED ISSUE #16: Does nothing for unknown execute extension

### LINKED ISSUE #17: No error for directory as executable

### PR DESCRIPTION
**Pull request by ronhippler**
close #17 
close #16

### CODE REVIEW

**Comment by eliassjogreen** [comment]
The windows formatting check failed for some reason but otherwise it looks good!

**Comment by ronhippler** [comment]
Do you know the syntax to check only on linux? 
Or should i remove the check for now?
I filed an issue on deno denoland/deno#4709

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -15,4 +15,5 @@ jobs:
       - uses: denolib/setup-deno@master
         with:
           deno-version: 0.40.0
+      - run: deno fmt --check
```
Comment (eliassjogreen) [inline .github/workflows/test.yml:18]: add `if: matrix.os == 'ubuntu-latest'` under this to only run linux

Comment (eliassjogreen) [reply .github/workflows/test.yml:18]: or `if: matrix.os != 'windows-latest'`

**Pull request merged**